strings/Longest_common_prefix.cpp

```diff
@@ -0,0 +1,51 @@
+//  A C++ Program to find the longest common prefix
+#include<bits/stdc++.h>
+using namespace std;
+
+// A Utility Function to find the common prefix between
+// strings- str1 and str2
+string commonPrefixUtil(string& str1, string& str2)
+{
+    string result = "";
+    int len = min(str1.length(), str2.length());
+
+    // Compare str1 and str2
+    for (int i = 0; i < len; i++)
+    {
+        if (str1[i] != str2[i])
+            break;
+        result += str1[i];
+    }
+
+    return (result);
+}
+
+// A Function that returns the longest common prefix
+// from the array of strings
```

Comment:
This function does not detect the _longest_ common prefix. 
The output of this program is
> The longest common prefix is - gee
Accorfing to the desciption I would expect it to be 
> The longest common prefix is - geeks

It would make sense qualify the array as const, as writing into it is not necessary.